testimony.yaml

```diff
@@ -60,6 +60,8 @@ CaseComponent:
     - Hosts
     - ImageMode
     - Infobloxintegration
+    - Insights-Advisor
+    - Insights-Vulnerability
```

Comment:
@tpapaioa is there an accompanying Jenkins GitLab MR that adds these components?

Reply:
Not yet; we have 1-2 additional IoP test updates to merge and then we'll create the Jenkins CI pipeline.